hotline/species/views.py

```diff
@@ -5,11 +7,25 @@
 from django.views.generic import CreateView, DeleteView, ListView, UpdateView
 
 from hotline.perms import permissions
+from hotline.species.forms import SpeciesSearchForm
 from hotline.species.models import Category, Severity, Species
 
 
-class SpeciesList(ListView):
-    model = Species
+def list_(request):
+    tab = request.GET.get('tabs') if request.GET.get('tabs') is not None else "search"
+    form = SpeciesSearchForm(request.GET, user=request.user)
+    species = form.results(request.GET.get("page", 1))
+
+    species_json = []
```

Comment:
What's this species_json variable for? I see you pass it to the species/list.html template, but I don't see it used in there.